lib/actions/api.js

```diff
@@ -77,8 +81,11 @@ function getActiveItinerary (state) {
  * that are no longer contained in the state we don't confuse the search IDs
  * with search IDs from the new session. If we were to use sequential numbers
  * as IDs, we would run into this problem.
+ *
+ * The updateSearchInReducer instructs the reducer to update an existing search
+ * if it exists. This is used by the field trip module.
  */
-export function routingQuery (searchId = null) {
+export function routingQuery (searchId = null, updateSearchInReducer = false) {
   return async function (dispatch, getState) {
```

Comment:
This `async` keyword looks to be unnecessary.

```suggestion
  return function (dispatch, getState) {
```

Reply:
Removed in a4480936984a2a1cb2d9dd6f97adc4ecd3a9a12c